presqt/api_v1/views/resource.py

```diff
@@ -1,11 +1,17 @@
+import os
+import shutil
+import bagit
```

Comment:
🥖

Reply:
Should be a line break between packages in the standard library vs. 3rd party stuff.